CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+hunker or take superior cover. The effect lasts for one out-of-combat 
+conversation or 5 turns in combat.
+
+==Transmission Vector ==
+Adrenaline Cost:60 Points
+Needle Cost: 1 Needle
+Duration: 	2 Actions
+Range:		5m radius centered at a point up to 30m from the Doc
+Effect:		You add an aerator to the end of your needle and launch it not at 
+one target in particular, but among the fray hoping to hit multiple targets.
+Targets without gas masks have the poison payload of your choice applied to 
+them.
+
+==Poison:	NF-40 ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	two weeks
+Range:		Payload (use in combination with other powers)
+    The biologic agent eats away at the target's flesh and biologic parts at an 
+astonishing rate. The target takes 20 + 2d10 damage per turn for the duration of 
+their affliction. They can be cured by decontamination devices or specially 
+trained medics. Targets that are wearing a gas mask, but have any exposed 
+biology are affected by this even if a deployment method says they wouldn't. 
+Use of biologic weapons are severely frowned upon in most civilized regions of 
+space.
+
+==Poison:	Lethal Toxin ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	almost instantaneous
+Range:		Payload (use in combination with other powers)
+    A biologic target afflicted with this poison must succeed on a DC 110 FORT 
+save or the target takes 180 damage that ignores shields and armor and silences 
+the target. If the target succeeds on their save, they take 60 damage instead. A 
+silenced target is prevented from using nanite abilities for 10 turns. (Also 
+prevents Engineering Processes and Doctor Procedures due to violent tremors). At 
+level 13, damage increases to 220.
+
+==Plague ==
+Adrenaline Cost: 250 Points
+Needle Cost: 2 Needles
+Duration: 	1 day
+Range:		40m
+    The target has the poison payload of your choice applied to them. Any time 
+another biologic target (friend or foe) comes within 8m of a target infected 
+with plague that new target gets infected too. This effect is recursive without 
+limit, but a target that has made a save against the payload delivered by this 
+procedure is inoculated from subsequent recursions of this procedure. If a 
+Poison Payload does not list a Save, use a DC 80 Fortitude save on spreading the
+plague. A target which has succeeded on their save or a target that has died is
+not contagious anymore.
+
+==Mild Phenethylamine Agonist ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagions
+Duration: 1 Action
+Range: Applies to the Doc only
+    For 3 turns, the doc gets a -3 to miss with guns, grenades, spells, and 
+melee weapons. They also get a +30 to reload and a +1 chance to crit.
+
+==Free Running ==
+Adrenaline Cost: 60 Points
+Nanite Cost: 15 Nanites
+Duration: 	2 Actions
+Range:		5m radius centered at a point up to 30m from the Doc
+    Your nanites form microscopic bonds with any solid surface allowing 
```

Comment:
"nanites" -> "Nanites"

FYI I've been capitalizing "nanite/nanites" throughout all of the CX rules. But I'm unsure if we actually do need to capitalize it. I've been treating it like a proper noun, but come to think of it, I don't know if it is. I don't think there is a dictionary definition of "nanite/nanites" yet.

Reply:
Good Call. Will Ctrl+H it.